.cicd/pipeline.yml

```diff
@@ -518,6 +518,19 @@ steps:
       queue: "automation-eks-eos-tester-fleet"
     timeout: 10
 
+  - label: ":beer: Brew Updater"
+    command: |
+      buildkite-agent artifact download eosio.cdt.rb . --step ':darwin: Mojave - Package Builder'
+      mv eosio.cdt.rb eosio_cdt_mojave.rb
+      buildkite-agent artifact upload eosio_cdt_mojave.rb
+      buildkite-agent artifact download eosio.cdt.rb . --step ':darwin: Catalina - Package Builder'
+      mv eosio.cdt.rb eosio_cdt_catalina.rb
+      buildkite-agent artifact upload eosio_cdt_catalina.rb
+    agents:
+      queue: "automation-basic-builder-fleet"
+    timeout: "${TIMEOUT:-5}"
+    skip: ${SKIP_MACOS_10_14}${SKIP_MACOS_10_15}${SKIP_PACKAGE_BUILDER}
```

Comment:
Ewww, our `SKIP` variables are inconsistent across branches?!?!?!?!?!?

Reply:
Yep, I left the old branch as it was... sorry.